src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NodeFactory.cs

```diff
@@ -407,7 +409,7 @@ private void CreateNodeCaches()
                 if (CompilationModuleGroup.ShouldProduceFullVTable(type))
                     return new EagerlyBuiltVTableSliceNode(type);
                 else
-                    return new LazilyBuiltVTableSliceNode(type);
+                    return _vtableSliceProvider.GetSlice(type);
```

Comment:
I'll have a look into that once this flows to the TFS side. This is best addressed from there to avoid unnecessary build breaks.